### DISCUSSION TITLE
The Cloudsmith site seems to have empty binaries

### DISCUSSION THREAD

**Discussion by dnghiem** [bareboat-necessities/lysmarine_gen#398] [Q&A] [answered]
Hi all, 

I am interested in getting the binary of the package, but everytime I try to save it from the Cloudsmith site, whether I clink on it or I save link as, it downloads a zero byte file. Are the binaries uploaded, or am I missing something here?

**Comment by mgrouch** [comment, marked as answer]
> Hi all,
> 
> I am interested in getting the binary of the package, but everytime I try to save it from the Cloudsmith site, whether I clink on it or I save link as, it downloads a zero byte file. Are the binaries uploaded, or am I missing something here?

Use instructions from the release notes to download from GitHub using bash script. 

https://github.com/bareboat-necessities/lysmarine_gen/releases/tag/v2024-03-14

Cloudsmith is out of free quotas for the rest of this month due to popular demand for downloads.
